notification/k8s/values.yaml

```diff
@@ -0,0 +1,40 @@
+fullnameOverride: 'ctp-adyen-integration-service-notification'
+
+image:
+  repository: 'eu.gcr.io/professionalserviceslabs/ctp-adyen-integration-notification'
+  tag: '1.1'
+
+service:
+  type: 'NodePort'
+
+# More on resources:
+# https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/resource-qos.md
+# https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+# https://cloud.google.com/compute/docs/machine-types
+
+# When the pod is scheduled, the container is guaranteed the amount of resources requested 2GiB RAM and 300m CPU time and it is not allowed to exceed 2GB RAM and 300m of CPU time.
+# This classifies the pod as "Guaranteed"
+#resources:
+#  requests:
+#    cpu: '300m' #30% of CPU time on 1 GCP Core
+#    memory: '1Gi' #equivalent to 2GiB
+#  limits:
+#    cpu: '300m' #30% of CPU time on 1 GCP Core
+#    memory: '1Gi' #equivalent to 2GB
```

Comment:
Why no resources requests/limits are set?

Reply:
We are in a faze of figuring it out 🙂